src/wagtailtrans/wagtail_hooks.py

```diff
@@ -1,5 +1,8 @@
 from django.conf.urls import include, url
-from django.contrib.staticfiles.templatetags.staticfiles import static
+try:
+    from django.contrib.staticfiles.templatetags.staticfiles import static
```

Comment:
Since we're supporting django >= 1.11 (at least the test suite) I think we don't need this ``try.. except`` block

https://github.com/django/django/blob/stable/1.11.x/django/templatetags/static.py

Reply:
Good catch, thanks!